packages/fluentui/e2e/cypress.json

```diff
@@ -0,0 +1 @@
+{ "video": false, "screenshotOnRunFailure": false, "integrationFolder": "./tests", "testFiles": "**.spec.ts" }
```

Comment:
I guess it would require extra work to dump them into a pipeline artifact and configuration etc...

Let's start without them then and see how it goes